src/shared/components/adminTabs/FlashRound/FlashRoundAdmin.tsx

```diff
@@ -79,14 +79,46 @@ const FlashRoundAdmin = () => {
 
 
     const saveChanges = () => {
-        FlashRoundService.update(game, {
-            ...flash,
-            answers1,
-            answers2,
-            type: 2,
-            currentRound: round,
-        })
-            .catch(error => console.error(error));
+
+        let allGood = 1;
+        let sumPoints = 0;
+
+        if (round == 1){
+            for (let i=0; i<answers1.length; i++){
+                sumPoints += answers1[i].points
+                if (!(answers1[i].answer.length > 1 && answers1[i].answer.length<50 && answers1[i].points >=0 && answers1[i].points < 101)){
+                    allGood = 0;
```

Comment:
Let's remove the check for maximum points. It is alright with 0 points, but we should not restrict the users to enter a maximum of 100 points